spec/features/admin_can_create_new_item_spec.rb

```diff
@@ -5,7 +5,8 @@
   context "as an admin" do
     it "can create a new item" do
       admin = create(:admin_user)
-      allow_any_instance_of(ApplicationController).to receive(:current_user).and_return(admin)
+      allow_any_instance_of(ApplicationController).to receive(:current_user)
+                                                  .and_return(admin)
```

Comment:
Place the . on the previous line, together with the method call receiver.<br>Use 2 (not 44) spaces for indenting an expression spanning multiple lines.